sylius/resource-bundle/1.9/manifest.json

```diff
@@ -0,0 +1,11 @@
+{
+    "container": {
+        "locale": "en"
```

Comment:
Why is this needed?

Reply:
Legacy. The bundle still uses this parameter as the default locale for translations instead of using configuration. 🙈

Reply:
@vvasiloi Maybe we can fix it before RC release. But I think we have to mention it on UPGRADE file, cause it can break the configuration.

Reply:
Yes, we could deprecate it. Not sure how to configure it though. Should we define new config, or use the config of Symfony Translations? Right now it assumes that parameter is used to configure Symfony Translations.

Reply:
@loic425 we're waiting for https://github.com/Sylius/SyliusResourceBundle/pull/374, right?

Reply:
Yes!